### DISCUSSION TITLE
Can GitHub make an AI copilot to summarize issue for users?

### DISCUSSION THREAD

**Discussion by JadeRay** [community/community#77916] [Copilot Conversations] [answered]
An AI Copilot that can summarize an issue for users. So we can find explicit solutions from the summary quickly. I think this is helpful.

**Comment by max0700** [comment]
An AI Copilot that can summarize an issue for users. So we can find explicit solutions from the summary quickly. I think this is helpful.

**Reply by ayim** [reply, marked as answer]
If you have access to Copilot Chat in GitHub.com, you have access to a chat interface in your web browser that may be able to help you summarize issues

https://docs.github.com/en/enterprise-cloud@latest/copilot/github-copilot-enterprise/copilot-chat-in-github/using-github-copilot-chat-in-githubcom#asking-a-question-with-a-specific-context
https://docs.github.com/en/copilot/github-copilot-enterprise/copilot-chat-in-github/about-github-copilot-chat#proposing-code-fixes

**Comment by max0700** [comment]
An AI Copilot that can summarize an issue for users. So we can find explicit solutions from the summary quickly. I think this is helpful.

**Comment by samus-aran** [comment]
Hi @JadeRay , Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we're glad you're here! 

We want to make sure you’re getting the best support you can now that GitHub Universe has ended so I've gone ahead and moved this to the most relevant category for you.

Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by userException** [comment]
Check this web app which summarizes issue description,  comments, community reactions and extracts key links from the github issue url:

[gitbrief.cleancodestack.com](https://gitbrief.cleancodestack.com/)
